go/vt/vttablet/tabletserver/query_engine.go

```diff
@@ -470,7 +470,7 @@ func (qe *QueryEngine) handleHTTPQueryPlans(response http.ResponseWriter, reques
 	response.Header().Set("Content-Type", "text/plain")
 	response.Write([]byte(fmt.Sprintf("Length: %d\n", len(keys))))
 	for _, v := range keys {
-		response.Write([]byte(fmt.Sprintf("%#v\n", utils.TruncateQuery(v))))
+		response.Write([]byte(fmt.Sprintf("%#v\n", utils.TruncateQuery(v, 512))))
```

Comment:
My only thought on that is that they are somewhat orthogonal -- truncating in the UI isn't actually affecting errors so I'd want to name it something different.